changelog.d/18521_add_main_separator_in_kubernetes_logs.md

```diff
@@ -0,0 +1 @@
+Added MAIN_SEPARATOR as a separator for parse_log_file_path in the kubernetes_logs source. Parsing paths for different OS has become correct.
```

Comment:
```suggestion
The `kubernetes_logs` source now handles parses log file paths on Windows to extract metadata.

authors: tmb-piXel
```

The changelog entries are meant to be user-facing rather than developer-facing so we just describe the user impact.